build.cake

```diff
@@ -402,6 +404,143 @@ Task("PackageConsole")
         });
     });
 
+Task("PackageChocolatey")
+	.Description("Creates chocolate packages of the console runner")
+    .IsDependentOn("CreateImage")
+	.Does(() =>
+	{
+		var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
+		
+		EnsureDirectoryExists(PACKAGE_DIR);
+		
+		// Note: Since cake does not yet support a working directory and separate output directory for chocolatey, the following copying and hacks are needed.
+		
+		// List with the addins (addin name, version, primary dll, additional files)
+		var addins = new Tuple<string, string, string, string[]>[] {
+			new Tuple<string, string, string, string[]>(
+				"NUnit.Extension.VSProjectLoader",
+				"3.5.0",
+				"vs-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string, string[]>(
+				"NUnit.Extension.NUnitProjectLoader",
+				"3.5.0",
+				"nunit-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string, string[]>(
+				"NUnit.Extension.NUnitV2ResultWriter",
+				"3.5.0",
+				"nunit-v2-result-writer.dll",
+				null
+			),
+			new Tuple<string, string, string, string[]>(
+				"NUnit.Extension.NUnitV2Driver",
+				"3.6.0",
+				"nunit.v2.driver.dll",
+				new [] {
+					"nunit.core.dll",
+					"nunit.core.interfaces.dll"
+				}
+			),
+			new Tuple<string, string, string, string[]>(
+				"NUnit.Extension.TeamCityEventListener",
+				"1.0.2",
+				"teamcity-event-listener.dll",
+				null
+			)
```

Comment:
@ChrisMaddock This came out of my directions to @Roemer. To the extent that we might build and distribute extensions written by other people, I thought we needed to be able to specify the version we have tested. RIght now, the only such extension is teamcity, which we don't test in any version. Now that I write that, it sounds bad, doesn't it. 😟